diff --git a/dist/node/chunks/node.js b/dist/node/chunks/node.js
index b28e858ffbffb78c18d53229dcc6cda1e2e216fb..ce6a5d8cb81a63dd4df6803c5ae3dddc9e06678e 100644
--- a/dist/node/chunks/node.js
+++ b/dist/node/chunks/node.js
@@ -12273,6 +12273,9 @@ async function startBrowserProcess(browser, browserArgs, url, logger) {
 			name: browser,
 			arguments: browserArgs
 		} } : {};
+		console.log(options);
+		console.log(url);
+		console.log("*****************");
 		new Promise((_, reject) => {
 			open(url, options).then((subprocess) => {
 				subprocess.on("error", reject);
